source/install/cloud-changelog.md

```diff
@@ -4,12 +4,59 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
 - [Release 2022-11-10](#release-2022-11-10)
 - [Release 2022-10-27](#release-2022-10-27)
 - [Release 2022-10-20](#release-2022-10-20)
-- [Release 2022-10-13](#release-2022-10-13)
+
+## Release 2022-12-01
+
+### Highlights
+
+#### Message priority and Acknowledgments
+ - Added message priority labels to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.
+
+#### Global Drafts
+ - Added a centralized page for draft messages.
+
+### Improvements
+
+#### User Interface (UI)
+ - Removed video check to allow the browser to decide what video types it can play.
+ - Added a tooltip to the right-hand side files filter icon.
+ - The number of users that can be added to a user group at once was increased to 256.
+ - Pre-packaged Boards v7.5.2.
+
+#### Administration
+ - Updated MacOS supported version to 11+.
+ - **My Insights** and OpenId Connect were added to the Free plan.
+ - All integration limits and usage limit components were removed.
+ - A notice banner in the **Invite** modal and at **system console > Site Statistics > Total Active Users** is now displayed to admins when the instance is in overage users state.
```

Comment:
```suggestion
 - A notice banner is displayed to admins on the **Invite** modal and on **System Console > Site Statistics > Total Active Users** page when the workspace exceeds the maximum number of users allowed.
```

Reply:
I'm not 100% sure my proposed update captures the correct state of the workspace for notification banners to appear based on "when the instance is in overage users state". This proposed update will benefit from a technical review.